src/main/resources/JIRA/createIssueRequestBody.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
These files won't exist when users use SHAFT as a dependency, so SHAFT won't be able to read them.

I recommend creating a class that holds both requests as simple strongs with getters so you can get and later replace the values you need.

With the latest JAVA you can actually use three double quotes, and this will create something called multiline string, try it... You'll like it... You can paste the json content inside it and jt will work.

Lastly it's a good practice to type ${VARIABLE_NAME} all in upper snake case and between curly brackets with the dollar sign as the placeholders for the text that you intend to change. It's better for readability so that people who read the file or string can immediately know that you intend to replace these substrings.

Reply:
Yeah, I really loved the three double quotes tip :D 
Thank you Mohab for your valuable comments. all changes applied.